.readthedocs.yml

```
@@ -3,7 +3,7 @@ version: 2
 build:
     os: ubuntu-22.04
     tools:
-        python: '3.8'
+        python: '3.9'
```

Why not just use python3.13? Then we do not to bother when deprecate python3.9.

---

just tried to be consistent using the lowest version, but I don' mind
```suggestion
        python: '3.13'
```